controllers/follow.js

```diff
@@ -0,0 +1,82 @@
+
+/**
+ * Controller for follow and unfollow
+ */
+export default class FollowController {
+  /**
+   * @description follow a user
+   * @param {object} req http request object
+   * @param {object} res http response object
+   * @returns  {object} response
+   */
+  static async followUser(req, res) {
+    const { userFollowing, userToFollow, userToFollowId } = req;
+    // Check if user is already following the user and return an error if they are
+    const getFollowing1 = await userFollowing.getFollowing();
+    let userIsFollowing = false;
+    getFollowing1.forEach((element) => {
```

Comment:
@youngestdj is there a better way to do this? If the user has 1 million followers, it means we're gonna loop through them all just to find if one exists.
There's a way to do this but I'm not sure if Sequelize offers that. See if you can get an array of `ids` for all the followers via a Sequelize method, then you can just do an `includes` to find if the id exists. cc @leksyib you can check if this exists in Sequelize too.

Reply:
Alright @9jaswag

Reply:
I have used an inbuilt Sequelize function to achieve this